joomla/joomla-cms
# [4.0] Add fallback for not complete manifest caches used in the pre-update checker

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/22421#issuecomment-531125432 reported by @infograf768 

### Summary of Changes

Add fallback for not complete manifest caches used in the pre-update checker

### Testing Instructions

.- install 4.0-dev
- install patchtester
- go to the database and clear the field `manifest_cache` for the patchtester record in the extensions table.
- set the [nightly major update server](https://update.joomla.org/core/nightlies/next_major_list.xml)
- notice the pre update checker
- notice that the pre update checker shows nothing as version for the patchtester and raises a notice
- apply this patch
- the updater now shows the string `Unknown version in this case and should not raise and notice any more.

### Expected result

No errors; After the patch the pre-update checker shows `Unknown version` before that info was just missing / empty.

### Actual result

```
Notice: Trying to get property 'version' of non-object in .../administrator/components/com_joomlaupdate/Model/UpdateModel.php on line 1357
```

And an empty version info for that extension.

### Additional steps

- [ ] [3.10-dev needs to be updated too.](https://github.com/joomla/joomla-cms/blob/3.10-dev/administrator/components/com_joomlaupdate/models/default.php#L1387)
